src/main/java/primeNumbers/PrimeNumbersDefinition.java

```diff
@@ -0,0 +1,23 @@
+package primeNumbers;
+
+import java.util.*;
+
+public class PrimeNumbersDefinition {
+    private int numbersCounter = 0;
+    private List<Integer> primeNumbersList = new ArrayList<>();
+
+    public List<Integer> getPrimeNumbers(int to) {
+        for (int i = 2; i <= to; i++) {
+            for (int j = i; j > 0; j--) {
+                if (i % j == 0) {
+                    numbersCounter++;
+                }
+            }
+            if (numbersCounter <= 2) {
+                primeNumbersList.add(i);
+            }
+            numbersCounter = 0;
```

Comment:
it looks like that could be optimized in some way.... also let's try to rewrite it in a functional way

Reply:
Optimized in e10c88a644d8e317f6eebafd5b061a0e6df590d7.